sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/TagName.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.clienttelemetry;
+
+import java.util.Locale;
+
+public enum TagName {
+    Container("Container", 1),
+    Operation("Operation", 2),
+    OperationStatusCode("OperationStatusCode", 4),
+    ClientCorrelationId("ClientCorrelationId", 8),
+    ConsistencyLevel("ConsistencyLevel", 16),
+    PartitionKeyRangeId("PartitionKeyRangeId", 32),
+    RequestStatusCode("RequestStatusCode", 64),
+    RequestOperationType("RequestOperationType", 128),
+    RegionName("RegionName", 256),
+    ServiceEndpoint("ServiceEndpoint", 512),
+    ServiceAddress("ServiceAddress", 1024),
+    IsForceRefresh("IsForceRefresh", 2048),
```

Comment:
maybe using 1 << 1, 2 etc is easier to understand here - thinking about if we want to add more in the future, the number may become super large

Reply:
Changed - I don't care much - my preference was readability (assuming not everyone might know immediately about hte shift operator <<) - but given that the pattern is obvious I assume we can expect folks to understand that.